### PR TITLE
Added pytest_saas as sub project

### LINKED ISSUE #9: Add sub-project for exasol-saas-api
Project [exasol-saas-api](https://github.com/exasol/saas-api-python) provides a generated client API enabling Python applications connecting to Exasol database SaaS instances and using their SaaS services.

For convenient usage in other projects, this tickets requests to add a plugin to the current project offering some fixtures etc..

### PR DESCRIPTION
**Pull request by ckunki**
closes #9

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -12,10 +12,9 @@ test +projects=PROJECTS:
         poetry -C ${p}/ run nox -f ${p}/noxfile.py -s coverage
     done
 
-
 # Create a release
 release project version:
-    @echo "This currently is a stub, in the future the workflow probably will look somthing like this:"
+    @echo "This currently is a stub, in the future the workflow probably will look something like this:"
```
Comment (Nicoretti) [inline justfile:17]: I think the `echo`s can be removed they have been there as dummy until replaced with an actual implementation.

File: pytest-itde/README.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
-pytest-itde Plugin
-==================
+pytest-exasol-itde Plugin
```
Comment (Nicoretti) [inline pytest-itde/README.rst:1]: This heading does not seem to be consistent with the one in saas:

`pytest-exasol-itde Plugin`
vs
`pytest-saas Plugin`

File: pytest-saas/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# pytest-saas Plugin
```
Comment (Nicoretti) [inline pytest-saas/README.md:1]: see previous comment about the consistency.

File: pytest-saas/doc/changes/unreleased.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Unreleased
+
+## Summary
+
+🚀 Initial Release of the pytest-saas Plugin
+
+This version introduces the `pytest-saas` plugin providing pytest functionalities for the API to Exasol SaaS instances provided by [exasol-saas-api](https://github.com/exasol/saas-api-python).
```
Comment (Nicoretti) [inline pytest-saas/doc/changes/unreleased.md:7]: `pytest-exasol-saas`?

File: justfile (hunk 1 above)
Comment (ckunki) [reply justfile:17]: I was not sure whether we should prefer bare calls to poetry or nox. 
As there is also nox session prepare-release.

But see next push for removed echo commands.

File: pytest-itde/README.rst (hunk 2 above)
Comment (ckunki) [reply pytest-itde/README.rst:1]: Thanks. See next push with updated file `pytest-saas/README.md` consistently using `pytest-exasol-saas`.

File: pytest-saas/doc/changes/unreleased.md (hunk 4 above)
Comment (ckunki) [reply pytest-saas/doc/changes/unreleased.md:7]: Yes, thanks, fixed in next push,

**Review by Nicoretti — APPROVED** [review]

**Pull request merged**